packages/gotrue/lib/src/gotrue_client.dart

```diff
@@ -526,9 +526,13 @@ class GoTrueClient {
     );
   }
 
-  ///Resends an existing signup confirmation email, email change email, SMS OTP or phone change OTP.
+  /// Resends an existing signup confirmation email, email change email, SMS OTP or phone change OTP.
   ///
-  ///Use [EmailResendParams] or [PhoneResendParams] to specify the type of resend.
+  /// Use [EmailResendParams] or [PhoneResendParams] to specify the type of resend.
```

Comment:
```suggestion
```
This line can be removed as it's a wrong leftover of #517, because these classes don't exist